docs/ai-backends.md

```diff
@@ -0,0 +1,194 @@
+# AI Backends
+
+Wagtail Vector Index can be configured to use different backends to support different AI services.
+
+There are two types of backends available for customisation:
+
+-   Embedding Backends - used to generate vector representations of text
+-   Chat Backends - used to generate 'chat' responses from queries
+
+Currently the only (and default) backends available in Wagtail Vector Index are `LLMChatBackend` and `LLMEmbeddingBackend`.
+
+Backends and their associated settings can be customised using the `WAGTAIL_VECTOR_INDEX` setting in your Django project settings file:
+
+```python
+WAGTAIL_VECTOR_INDEX = {
+    "CHAT_BACKENDS": {
+        "default": {
+            "CLASS": "wagtail_vector_index.ai_utils.backends.llm.LLMChatBackend",
+            "CONFIG": {
+                "MODEL_ID": "gpt-3.5-turbo",
+            },
+        },
+    },
+    "EMBEDDING_BACKENDS": {
+        "default": {
+            "CLASS": "wagtail_vector_index.ai_utils.backends.llm.LLMEmbeddingBackend",
+            "CONFIG": {
+                "MODEL_ID": "ada-002",
+            },
+        }
+    },
+}
+```
+
+## Chat Backends
+
+### `LLMChatBackend`
+
+This backend uses the ["LLM" library](https://llm.datasette.io/en/stable/) which offers support for many AI services through plugins.
+
+By default, it is configured to use OpenAI's `gpt-3.5-turbo` model.
```

Comment:
**Question:** Is this statement true?

Reply:
Yeah, in testing and documented here https://llm.datasette.io/en/stable/setup.html#setting-a-custom-default-model